clr/config.py

```diff
@@ -2,6 +2,11 @@
 from pathlib import Path
 import os.path
 import os
+import runpy
+
+"""The clr tool is configred using a python source file (clrfile.py) that is
```

Comment:
Moving this docstring above the imports will make it the value of `__doc__` for the module (where it is now, it's just an unused string literal).

Reply:
Can also be explicit by assigning via `__doc__ = `.

Reply:
Yup, that definitely does work, but [docstring convention](https://www.python.org/dev/peps/pep-0257/) is to just put the string literal at the top of the file:

> A docstring is a string literal that occurs as the first statement in a module, function, class, or method definition. Such a docstring becomes the `__doc__` special attribute of that object.

In general, I think it's best to just let the built-in parts of the language handle interacting with dunder values where possible. (e.g. `a.__add__(b)` works, but one can also just do `a + b`).